### PR TITLE
FINERACT-1761: Loan Product repayment due event values set by default…

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
… from global configurations

## Description

As part of Loan Product enhancements in the create and edit actions some values are being used from the Global Configuration. In particular the Event (notifications) settings values, so those values must to be remarked to indicated the default value is set from the GLobal Configuration  

[FINERACT-1761](https://issues.apache.org/jira/browse/FINERACT-1761)

Screenshot 2023-10-29 at 5 52 18 p m

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by josehernandezfintecheandomx** [comment]
TLDR... Not needed it more

**Comment by josehernandezfintecheandomx** [comment]
TLDR... Not needed it more
